tests/unit/utils/test_utils.py

```diff
@@ -122,3 +123,16 @@ def test_makedirs(repo_dir):
 
     makedirs(path_info)
     assert os.path.isdir(path_info.fspath)
+
+
+def test_is_child_path(repo_dir):
+    child = os.path.join("path", "to", "folder")
+    assert not is_child_path(child, parent=child)
```

Comment:
Does this come as confusing?

Reply:
Not really :) But maybe consider using @pytest.mark.parametrize instead of copypasting the assert 🙂